client/ayon_core/plugins/publish/collect_settings.py

```diff
@@ -9,4 +9,10 @@ class CollectSettings(api.ContextPlugin):
     label = "Collect Settings"
 
     def process(self, context):
-        context.data["project_settings"] = get_current_project_settings()
+        project_name = context.data["projectName"]
+        self.log.debug(
+            "Collecting settings for project: {}".format(project_name)
+        )
+        project_settings = get_project_settings(project_name)
+        context.data["project_settings"] = project_settings
+        context.data["projectSettings"] = project_settings
```

Comment:
What's the reason this key got added? Is `project_settings` deprecated?

Should we add that key in a separate PR if our intent is to swap the usage around - and in that other PR also refactor all usages to match the newer key if that's the preferred key name. Then just keep `project_settings` for backwards compatibility and mark it with a comment as such.

I feel like having the duplicate key is worse than having just one key that may not be named the way we want it.

Reply:
```suggestion
```

Reply:
100% different PR.